Expensify/App
# Remove "Comment Deleted" message

### Details
Simply deletes a comment instead of showing `[Comment deleted]`

### Fixed Issues
$ https://github.com/Expensify/App/issues/7646

### Tests
1. Pull the [Auth PR](https://github.com/Expensify/Auth/pull/6430) changes and run `./script/makeAll.sh`
2. Send multiple messages to another user
2. Delete some messages and notice that the message completely disappears
3. Delete the last message sent and verify that the last message in the LHN is updated accordingly
4. Refresh the app and notice that the last message in the LHN is correct

**Note:** if the LHN message does not update, you might have to `make clean` and run `./script/makeAll.sh` again.


### QA Steps
Steps 2-4 above.


### Tested On


### Screenshots


#### Web

https://user-images.githubusercontent.com/22219519/154084654-9cbf8a6f-ae2b-475a-840d-52f00f5028a5.mov

#### Mobile Web

https://user-images.githubusercontent.com/22219519/154093118-fbfddae0-04b9-45b3-8eaa-310652136977.mov

#### Desktop

https://user-images.githubusercontent.com/22219519/154084697-7c4482a0-967a-4edd-b03a-544a50d6f899.mov

#### iOS

https://user-images.githubusercontent.com/22219519/154093155-6925e5c7-e5a9-4c79-89c0-6e95c7af4975.mov

#### Android

https://user-images.githubusercontent.com/22219519/154094010-c5018f1c-ceca-4676-8d7d-5ba2326e2c53.mov
